pages/resources/quests.mdx

```diff
@@ -343,21 +356,24 @@ Retrieves the reward code for the specified platform and returns a [quest reward
 
 ###### JSON Params
 
-| Field    | Type    | Description                                                   |
-| -------- | ------- | ------------------------------------------------------------- |
-| location | integer | Used for analytics (probably)                                 |
-| platform | string  | The [platform to retrieve the code for](#quest-platform-type) |
+| Field    | Type    | Description                                                                |
+| -------- | ------- | -------------------------------------------------------------------------- |
+| location | integer | The [content location](#quest-content-type) where the action was initiated |
+| platform | string  | The [platform](#quest-platform-type) to retrieve the code for              |
 
 <RouteHeader method="POST" url="/quests/{quest.id}/heartbeat">
   Send Quest Heartbeat
 </RouteHeader>
 
 Tells the server to update the `last_stream_heartbeat` and `stream_progress_seconds` fields. Used for keeping track of how long the stream has been running for, and for checking if the user has met the [stream duration requirement](#quest-config-structure).
 
+The `stream_progress_seconds` field is incremented by the amount of seconds since the last heartbeat request, up to 2 minutes at once.
+
 Returns a [quest user status](#quest-user-status-object) object.
 
 ###### JSON Params
 
-| Field      | Type   | Description                                                                                           |
-| ---------- | ------ | ----------------------------------------------------------------------------------------------------- |
-| stream_key | string | The encoded key of the stream (e.g `guild:169256939211980800:1050497861969793164:222069018507345921`) |
+| Field      | Type    | Description                                                                                           |
+| ---------- | ------- | ----------------------------------------------------------------------------------------------------- |
+| stream_key | string  | The encoded key of the stream (e.g `guild:169256939211980800:1050497861969793164:222069018507345921`) |
+| terminal?  | boolean | Whether this is the last heartbeat in the current sequence                                            |
```

Comment:
Does this mean the last one cuz the stream ended? Or the last one cuz they got the reward

Reply:
Last one cuz the stream ended